### PR TITLE
[civetweb] Update to version 1.12

### PR DESCRIPTION
**Pull request by gjasny**
Update civetweb from snapshot release to regular 1.12 release. Tested locally on MacOS against prometheus-cpp.

### CODE REVIEW

File: ports/civetweb/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: civetweb
-Version: 2019-07-05-1
+Version: 1.12
```
Comment (Cheney-W) [inline ports/civetweb/CONTROL]: Could you please add `Homepage: https://github.com/civetweb/civetweb` to the next line of `Version: 1.12`？

File: ports/civetweb/portfile.cmake [hunk 2]
```diff
@@ -9,8 +9,8 @@ endif()
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO civetweb/civetweb
-    REF 2c1caa6e690bfe3b435a10c372ab2dcd14b872e8
-    SHA512 bfd37906f85c10649108f83e755f28f058c0c27b0d597e6eb82f097db7fa043f6014984f1735c904d0e01c8a5e0dc45f1c57c1fb45b08bce78f42539e19160d6
+    REF 4b440a339979852d5a51fb11a822952712231c23
```
Comment (Cheney-W) [inline ports/civetweb/portfile.cmake]: Could you please modify this line as `REF 4b440a339979852d5a51fb11a822952712231c23 # v1.12`?

**Review by Cheney-W — CHANGES_REQUESTED** [review]
Thanks for your PR! 
1. `include(vcpkg_common_functions)` is no longer needed, could you please remove this line?
2. Could you please modify
`if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")`
    `message(FATAL_ERROR "${PORT} does not currently support UWP")`
`endif()`
as below:
`vcpkg_fail_port_install(MESSAGE "${PORT} does not currently support UWP" ON_TARGET "UWP")`
3. Could you please modify
`configure_file(${SOURCE_PATH}/LICENSE.md ${CURRENT_PACKAGES_DIR}/share/civetweb/copyright COPYONLY)`
as below:
`file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)`

**Comment by gjasny** [comment]
Thanks for the review. I addressed all of your comments and re-tested locally.

**Comment by Cheney-W** [comment]
@gjasny 
Could you please remove the below lines from `vcpkg/scripts/ci.baseline.txt`?
`civetweb:x64-windows        = skip`
`civetweb:x64-windows-static = skip`
`civetweb:x86-windows        = skip`

**Comment by gjasny** [comment]
@Cheney-W done

**Comment by LilyWangL** [comment]
@gjasny Thanks for your PR. I have tested this PR on Linux and MACOS, and it passed. Please remove the below lines from `vcpkg/scripts/ci.baseline.txt`.
`civetweb:x64-linux          = skip`
`civetweb:x64-osx            = skip`

**Review by Cheney-W — APPROVED** [review]

**Comment by Cheney-W** [comment]
@gjasny Could you address Lily's suggestion? Thanks!

**Comment by gjasny** [comment]
Sorry, I must have missed the notification email.

**Comment by strega-nil** [comment]
Alright, this looks great! Thanks @gjasny :)

**Pull request merged**